src/main/java/karuna/karuna_backend/content/dto/ContentResponseDTO.java

```diff
@@ -0,0 +1,4 @@
+package karuna.karuna_backend.content.dto;
+
+public record ContentResponseDTO(long Id, String page, String key, String value) {
```

Comment:
No need to implement id field in ContentDTO, just stick to 1 DTO in that case.

Reply:
SO i can delete this record because is the same as a contentcreatedto